.env

```diff
@@ -1,3 +1,3 @@
-# REACT_APP_EGGHEAD_BASE_URL=https://egghead-io-staging.com
-REACT_APP_EGGHEAD_BASE_URL=http://egghead.dev:5000
+REACT_APP_EGGHEAD_BASE_URL=https://egghead-io-staging.com
```

Comment:
I'd like to remove `.env` entirely and just use yarn scripts instead; so `yarn dev` uses staging , `yarn dev:local`, `yarn dev:prod` etc